go.mod

```diff
@@ -161,7 +161,10 @@ require (
 	github.com/sijms/go-ora/v2 v2.2.22
 )
 
-require github.com/nats-io/nkeys v0.3.0
+require (
+	github.com/hashicorp/go-uuid v1.0.1
```

Comment:
Could we use `github.com/google/uuid` to reduce deps?

Reply:
fine